src/web/app/pages-instructor/instructor-home-page/__snapshots__/instructor-home-page.component.spec.ts.snap

```diff
@@ -0,0 +1,2741 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InstructorHomePageComponent should snap with default fields 1`] = `
+<tm-instructor-home-page
+  SessionsTableColumn={[Function Object]}
+  SessionsTableHeaderColorScheme={[Function Object]}
+  SortBy={[Function Object]}
+  courseService={[Function CourseService]}
+  courseTabModels={[Function Array]}
+  feedbackQuestionsService={[Function FeedbackQuestionsService]}
+  feedbackSessionsService={[Function FeedbackSessionsService]}
+  httpRequestService={[Function HttpRequestService]}
+  instructorCoursesSortBy={[Function Number]}
+  navigationService={[Function NavigationService]}
+  ngbModal={[Function NgbModal]}
+  route={[Function ActivatedRoute]}
+  router={[Function Router]}
+  statusMessageService={[Function StatusMessageService]}
+  studentSearchkey=""
+  timezoneService={[Function TimezoneService]}
+  user="undefined"
+>
+  <h1>
+    Home
+  </h1><div
+    class="card bg-light mb-4"
+  >
+    <div
+      class="card-body"
+    >
+      <div
+        class="form-group"
+      >
+        <div
+          class="input-group"
+        >
+          <input
+            class="form-control ng-untouched ng-pristine ng-valid"
+            ng-reflect-model=""
+            placeholder="e.g. Charles Shultz, charles@gmail.com"
+            type="text"
+          />
+          <div
+            class="input-group-append"
+          >
+            <button
+              class="btn btn-primary"
+            >
+              Search
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div><div
+    class="row height-60px"
+  >
+    <div
+      class="col-4"
+    >
+      <a
+        class="btn btn-primary text-white"
+        href="/web/instructor/courses"
+        ng-reflect-router-link="/web/instructor/courses"
+        routerlink="/web/instructor/courses"
+      >
+        Add New Course
+      </a>
+    </div>
+    <div
+      class="col-8"
+    >
+      <div
+        class="float-right"
+      >
+        <h5
+          class="d-inline"
+        >
+          <strong>
+             Sort By: 
```

Comment:
There is no courses here, so it doesn't make sense to have a "Sort by" displayed.